config/initializers/deprecations.rb

```diff
@@ -1 +1,29 @@
 ActiveSupport::Deprecation.behavior = :silence if Rails.env.production?
+
+Foreman::Application.configure do |app|
+  config.after_initialize do
+    unless Foreman.in_rake?
```

Comment:
```suggestion
    break if Foreman.in_rake?
```
will reduce indentation level :)